docs/user-guide/rules/no-disallowed-headers.md

```diff
@@ -143,6 +143,65 @@ Note that:
 
 </details>
 
+<details>
+<summary>How to configure IIS</summary>
+
+To configure what headers are add or removed, you can use the
+[`<customHeader> element`][customheader] and `<remove>/<add>`
+depending on what you need.
+
+The following snippet will remove the `X-Powered-By` headers from all
+responses:
+
+```xml
+<configuration>
+     <system.webServer>
+        <httpProtocol>
+             <customHeaders>
+                <remove name="X-Powered-By"/>
+             </customHeaders>
+         </httpProtocol>
+    </system.webServer>
+</configuration>
+```
+
+Removing the `Server` header is a bit more complicated and changes
+depending on the version.
+
+In IIS 10.0 you can remove it using the [`removeServerHeader` attribute of `requestFiltering`][requestfiltering]:
+
+```xml
+<configuration>
+     <system.webServer>
+        <security>
+            <requestFiltering removeServerHeader ="true" />
+        </security>
+    </system.webServer>
+</configuration>
+```
+
+For previous versions of IIS (7.0-8.5) you can use the following:
+
+```xml
+<configuration>
+     <system.webServer>
+        <rewrite>
+            <outboundRules rewriteBeforeCache="true">
+                <rule name="Remove Server header">
+                    <match serverVariable="RESPONSE_Server" pattern=".+" />
+                    <action type="Rewrite" value="" />
+                </rule>
+            </outboundRules>
+        </rewrite>
+    </system.webServer>
+</configuration>
+```
+
+The above snippet will use a [`URL rewrite`][url rewrite] rule to remove
```

Comment:
Does IIS have this module enabled by default or users usually need to enable it?

If it's always enabled by default, then I don't think we should specify this.